Reduce Sampling footprint in "Set Up Tracing" (#10505)
getsentry/sentry-docs: docs/platforms/javascript/common/tracing/index.mdx

While you're testing, set <PlatformIdentifier name="traces-sample-rate" /> to `1.0`, as that ensures that every transaction will be sent to Sentry. Once testing is complete, you may want to set a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switch to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions, based on contextual data.

### Sampling

> Leaving your sample rate at `1.0` means that traces will be started for every page load, navigation, and other events like function calls within your application. Depending on the amount of traffic your application gets, you may want to consider setting a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions.

Reviewer: ```suggestion
If you leave your sample rate at `1.0`, a trace will be started for every page load, navigation, and any other events like function calls within your application. Depending on the amount of traffic your application gets, you may want to consider setting a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions.
```
